french_law/js/examples.js

```diff
@@ -7,20 +7,20 @@ var suite = new Benchmark.Suite();
 
 function run_computation_AF(log) {
   var result = Law.computeAllocationsFamiliales({
-    iDateCouranteIn: new Date(Date.UTC(2020, 04, 20)),
+    iDateCouranteIn: "2020-04-20",
```

Comment:
I guess this is OK, and I know I had troubles with timezones in the past for building those `Date` objects with the right date. Plus in your app you can use https://bobbyhadz.com/blog/javascript-get-iso-date-without-time to convert a `Date` into an ISO string. But the fact that the ISO format is expected should be documented somewhere.

Reply:
I added a [comment](https://github.com/CatalaLang/catala/pull/296/files#diff-aeeaa5142d156c4a9cf8c11a2255791619420125c38d709c9d9a8887eaf45db7R108) in the JSOO runtime interface. But yea, I think it should be specified in the `french_law/js/README.md` too.